src/mixins/Themeable.ts

```diff
@@ -176,11 +179,11 @@ export function ThemeableMixin<T extends Constructor<WidgetBase<ThemeablePropert
 			const appliedClasses = classNames
 				.filter((className) => className !== null)
 				.reduce((appliedClasses: {}, className: string) => {
-					if (!this._baseClassesReverseLookup[className]) {
-						console.warn(`Class name: ${className} is not from baseClasses, use chained 'fixed' method instead`);
+					if (!this._baseThemeClassesReverseLookup[className]) {
+						console.warn(`Class name: ${className} is not from 'registeredBaseThemeClasses', use chained 'fixed' method instead`);
```

Comment:
could do with making this message more understandable to the end user (as they will not know of `registeredBaseThemeClasses`